scripts/scripts/combine_sentences.py

```
@@ -0,0 +1,72 @@
+import argparse
+import json
+import sys
+from pathlib import Path
+from typing import List, Dict, Any
+
+
+def get_parser() -> argparse.ArgumentParser:
+    """Configure command line argument parser."""
+    parser = argparse.ArgumentParser(
```

Pylint (3.3.7)

[warning] Redefining name 'parser' from outer scope (line 70)

(W0621)